tests/testswitch.nim

```diff
@@ -85,21 +84,22 @@ suite "Switch":
     check "Hello!" == msg
     await conn.close()
 
-    await allFuturesThrowing(done.wait(5.seconds), switch1.stop(), switch2.stop())
+    await allFuturesThrowing(
+      done.wait().withTimeout(5.seconds), switch1.stop(), switch2.stop()
+    )
```

Comment:
`done.wait().withTimeout(5.seconds)` is probably error and should be like it was before
`done.wait(5.seconds)`